### DISCUSSION TITLE
[Support]: Insecure: https://<public url>/api/stats

### DISCUSSION THREAD

**Discussion by doug62** [blakeblackshear/frigate#15625] [General Support] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

This URL does not require authentication, and returns sensitive data - this is a serious security breach:
https://<public url>/api/stats

### Steps to reproduce

curl https://<public url>/api/stats

### Version

0.14.0

### Frigate config file

```yaml
NA
```

### docker-compose file or Docker CLI command

```yaml
Kubernetes / NA
```

### Relevant Frigate log output

```shell
NONE
```

### Relevant go2rtc log output

```shell
NONE
```

### Operating system

Other Linux

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

NA

### Screenshots of the Frigate UI's System metrics pages

NA

### Any other information that may be helpful

NA

**Comment by hawkeye217** [comment]
I can't reproduce this, and every other API call requires authentication.

How do you have Frigate configured?

**Reply by doug62** [reply]
When you 'curl https://<public url>/api/stats' which does not include any headers what do you see?  I should see nothing, with a 401 but instead I get a list of all my cameras

**Reply by doug62** [reply]
curl https://publicurl.com/api/stats

**Reply by NickM-27** [reply]
What port are you using to access frigate?

**Reply by doug62** [reply]
It is in a k8 pod on 5000 but I expose it differently, when i 'curl http://localhost:5000/api/stats I get a list of my cameras, no auth headers provided, typically a call is made to an auth endpoint to obtain a token and then subsequent calls need a header like 'Authorization: Bearer xyz'

Should perhaps mention that I am the software architect of a top 5 bank....

**Reply by hawkeye217** [reply, marked as answer]
This is your issue. Port 5000 is the unauthenticated port. See the docs: https://docs.frigate.video/configuration/authentication

**Comment by doug62** [comment]
Whups! - OMG, THANKS!
